_posts/data_reuse_000458.md

```diff
@@ -0,0 +1,43 @@
+
+## The Original Study: Peering into the Anesthetized Brain
+In June 2023, Dr. Leslie Claar and colleagues from the Allen Institute published their groundbreaking findings in the journal _eLife_ in the article, “Cortico-thalamo-cortical interactions modulate electrically evoked EEG responses in mice”. Their approach was comprehensive: they stimulated mouse cortex while simultaneously recording with EEG and neuropixels probes, comparing brain activity during wakefulness and under isoflurane anesthesia.
+
+In awake mice, Claar's team observed a robust event-related potential (ERP) when they delivered pulsatile electrical stimulation to deep layers of the motor cortex. This ERP coincided with a distinct pattern of neural spiking, both locally in the motor cortex and in distant sensorimotor-related thalamic nuclei. The response followed a three-act play: initial excitation within 25 milliseconds, a period of quiescence for about 125 milliseconds, and finally, a strong rebound excitation before returning to baseline.
+When the mice were under anesthesia, the story changed dramatically. The ERP's amplitude and complexity plummeted in response to the same stimulation. While the initial excitation persisted in the motor cortex, it vanished in the thalamic nuclei. The rebound excitation, along with its corresponding ERP component at around 180 milliseconds, was significantly weakened.
+
+Based on these observations, Claar and colleagues proposed that interactions between the cortex and thalamus are responsible for the rebound excitation phase of the stimulus response. This rebound excitation, in turn, drives the second component of the ERP – a crucial signal of wakefulness.
+
+After their experiments, Claar and her team published their data on DANDI, ensuring that it would remain publicly accessible for any future researchers who wanted to use it.
+
+## A New Perspective: Zooming in on Synaptic Potentials
+Just a few months later, in November 2023, Dr. Richard Burman and colleagues at the University of Oxford wrote a paper that included reanalysis of this data. Their findings were published in the journal _Neuron_ with the article, “Active cortical networks promote shunting fast synaptic inhibition _in vivo_.” They also investigated the effects of anesthesia on brain activity, but narrowed their focus to the differential function of a specific receptor subtype: GABA<sub>A</sub>R.
+
+Burman's team used a technique called _in vivo_ gramicidin perforated patch-clamp recording, combined with optogenetic activation of GABAergic synaptic inputs. They compared the GABA<sub>A</sub> receptor equilibrium potential (E<sub>GABA<sub>A</sub>R</sub>) in mice during wakefulness and under urethane anesthesia.
```

Comment:
if it was possible to add references for canonical papers or sites describing those techniques in short (wikipedia?), some lame readers like me could appreciate.